AssetTracker/apps/companyusers/serializers.py

```diff
@@ -4,4 +4,4 @@
 class CompanyUserSerializer(serializers.ModelSerializer):
     class Meta:
         model = CompanyUser
-        fields = ('User', 'Company')
\ No newline at end of file
+        fields = ('user', 'company')
```

Comment:
To add newline to the end of the file

Reply:
> I can resolve the comments by working on the `pylintConfiguration` branch.
> There might be a few things I missed, but I did my best.

All good my friend, fixed everything you mentioned but I am not gonna touch the spaces around =, it really looks weird. Thanks for taking the time to review, great stuff.